README.md

```diff
@@ -1,180 +1,26 @@
-|                    |                                                                                                                                  | Scan-Rate: 25 Hz   |                 |
-| LMS4xxx-Family      | [e.g. 1091423](https://www.sick.com/de/de/mess-und-detektionsloesungen/2d-lidar-sensoren/lms4000/lms4111r-13000/p/p578044?ff_data) | 1 layer max. range: 3 m, ang. resol. 0,0833 [deg]| ✔ [stable]|
-|                    |                                                                                                                                  | Scan-Rate: 600 Hz   |                 |
-|                    |                                                                                                   | Opening angle: +/- 50 [deg]   |                 |
-| RMS3xx             | [8021530](https://cdn.sick.com/media/docs/4/04/504/Operating_instructions_RMS3xx_en_IM0075504.PDF)| Radar Sensor | ✔ [stable]|
 
-
-## IMU Support
-Devices of the MRS6xxx and MRS1xxx series are available with an optionally built-in IMU.
-Further information on the implementation and use of the experimental Imu support can be found on the [Imu page](doc/IMU.md).
 ##  Start Node
 
 Use the following command to start ROS node:
 
-- For MRS6124:
-```bash
-roslaunch sick_scan sick_mrs_6xxx.launch
-```
-
-- For MRS1104:
-```bash
-roslaunch sick_scan sick_mrs_1xxx.launch
-```
-
-- For LMS1104:
-```bash
-roslaunch sick_scan sick_lms_1xxx.launch
-```
-
-- For TiM240-prototype:
-```bash
-roslaunch sick_scan sick_tim_240.launch
-```
-
-- For TiM4xx-family:
-```bash
-roslaunch sick_scan sick_tim_4xx.launch
-```
-
-- For TiM5xx-family:
-```bash
-roslaunch sick_scan sick_tim_5xx.launch
-```
-
-- For TiM7xx-family (no safety scanner):
-```bash
-roslaunch sick_scan sick_tim_7xx.launch
-```
-
-- For TiM7xxS-family (safety scanner):
-```bash
-roslaunch sick_scan sick_tim_7xxS.launch
-```
-
-- For LMS1xx-family:
-```bash
-roslaunch sick_scan sick_lms_1xx.launch
-```
-
-- For LMS5xx-family:
-```bash
-roslaunch sick_scan sick_lms_5xx.launch
-```
-
-- For LMS4xxx-family:
-```bash
-roslaunch sick_scan sick_lms_4xxx.launch
-```
-
-- For NAV210:
-```bash
-roslaunch sick_scan sick_nav_2xx.launch
-```
-
-- For NAV245:
-```bash
-roslaunch sick_scan sick_nav_2xx.launch
-```
-
-- For NAV310:
-```bash
-roslaunch sick_scan sick_nav_3xx.launch
-```
-
-- For RMS3xx-family:
-```bash
-roslaunch sick_scan sick_rms_3xx.launch (under
-opment)
-```
-
-### Starting Scanner with Specific Ip Address
-To start the scanner with a specific IP address, the launch command can be used for most launch files as follows.
-The hostname is the ip-address of the scanner:
-
-```bash
-roslaunch <launch-file> hostname:=<ip-address>
-```
-e.g.
+- For TiM771S:
 ```bash
-roslaunch sick_scan sick_tim_5xx.launch hostname:=192.168.0.71
+rosrun sick_scan sick_generic_caller __name:=sick_tim_7xxS _hostname:=10.39.46.23

```

Comment:
what about the `scanner_type: sick_tim_7xxS` ?

Reply:
that's what the driver expects